libindy-crypto/docs/anoncreds-design.md

```diff
@@ -1,24 +1,29 @@
-# Shared anoncreds math in Indy Crypto
+# Anonymous Credentials (anoncreds) crypto support
 Initial version of anoncreds protocol was implemented as part of Indy SDK (https://github.com/hyperledger/indy-sdk). After some discussion and community requests we decided to move low level anoncreds crypto code to indy-crypto project. This should allow the crypto to be audited and shared with other projects.
 
 ## Main ideas
-1. Indy-crypto will not provide high level anoncreds protocol details and persistence. It will operate low level crypto entities only. 
+1. Indy-crypto will not provide high level anoncreds protocol details and persistence. It will support low level crypto entities only. 
 1. API defines 3 actors:
     * Issuer - trust source that provides credentials to prover
     * Prover - credentials owner that can proof and partially disclose the credentials to verifier
     * Verifier - party that wants to check that prover has some credentials provided by issuer
-1. The list of entities that indy-crypto operates:
-    * Claim Schema” - a list of attributes a Claim is based on
-    * “Claim Values” - values of attributes from Claim Schema (must be integers)
-    * “Claim Signature” - signed by the Issuer part of the Claim
-    * “Issuer public key” and “issuer private key”. Keys will contain 2 internal parts. One for signing primary claims and second for signing non-revocation claims. These keys are used to proof that claim was issued and doesn’t revoked by this issuer. Issuer keys have global identifier that must be known to all parties.
-    * “Revocation registry public” and “revocation registry private”. Internally them will contain revocation keys, accumulator and accumulator tails. Public part of revocation registry must be shared by issuer in trusted place and can be used to proof that concrete claim wasn’t revoked.
-    * “Master secret” - secret prover data that is used to proof that prover owns the claim. Prover blinds master secret be generating “blinded master secret” and “master secret blinding data” and sends “blinded master secret” to isser that uses “blinded master secret” in claim creation. It allows to use this claim by prover only.
-    * “Proof” is complex crypto structure created by proved over multiple claims that allows to proof that prover:
-      * Owns claims issued with specific issuer keys (identified by key id)
+1. The list of entities that indy-crypto operates on:
```

Comment:
Should we mention `claim_context (m2)` attribute here?

Reply:
It is not a core object so i dont think so